__tests__/app.test.js

```diff
@@ -1,54 +1,123 @@
+        expect(response.status).toBe(200);
+        expect(Array.isArray(topics)).toBe(true);
+      });
+  });
+  test("Results Should contain a property of slug and description", () => {
+    return request(app)
+      .get("/api/topics")
       .then((response) => {
         const topics = response.body.topic;
         topics.forEach((topic) => {
-          expect(topic.hasOwnProperty('slug'))
-          expect(topic.hasOwnProperty('description'))
-        })
-        expect(response.status).toBe(200)
-      })
-    })
-  })
-  describe('GET/api', () => {
-    test('returns status 200 and an object', () => {
-      return request(app).get('/api').then((response) => {
-        const endPoints = response.body.endPoint;
+          expect(topic.hasOwnProperty("slug"));
+          expect(topic.hasOwnProperty("description"));
+        });
         expect(response.status).toBe(200);
-        expect(typeof endPoints).toEqual('object');
       });
-    });
-    test('returns 200 and an object with the same structure as endpoint json file', () => {
-      return request(app).get('/api').then((response) => {
-        const endPoints = response.body.endPoint;
+  });
+});
+describe("GET/api", () => {
+  test("returns status 200 and an object", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        const endPoints = response.body.endPoints;
+        expect(response.status).toBe(200);
+        expect(typeof endPoints).toEqual("object");
+      });
+  });
+  test("returns 200 and an object with the same structure and content as endpoint json file", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        const endPoints = response.body.endPoints;
         expect(response.status).toBe(200);
         expect(endPoints).toEqual(endPointData);
       });
-    });
-  });
\ No newline at end of file
+  });
+});
+describe("GET/api/articles/:article_id", () => {
+  test("returns status 200 and an object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then((response) => {
+        const article = response.body.article;
+        expect(response.status).toBe(200);
+        expect(typeof article).toEqual("object");
+      });
+  });
+  test("returns status 200 and an article object with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then((response) => {
+        const article = response.body.article;
+        expect(response.status).toBe(200);
+        expect(article[0]).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  })
```

Comment:
Good work on these tests, some feedback:

- I don't think 2 tests are needed, just have 1 which checks everything. So 1 test which checks the status code, it's an object and all the key value pairs. The point of an integration test is that everything works end-to-end.
- You don't need do `expect(response.status).toBe(200);`, supertest gives you that functionality by doing `.expect(200)`
- It's good to see that you're checking that the article returned is exactly article 1 and all its contents, good work.